testing/055_markdown.md

```diff
@@ -1,6 +1,6 @@
 <!--
 // objective: test markdown parsing
-// check: md_055_markdown.xml
+// check: md_055__markdown.xml
```

Comment:
I think the original reference file (without the double underscope) can be removed, right?

Reply:
Yes, I had some trouibles with git and this one was, unfortunately, left over.
I removed it.